### PR TITLE
update example.html to include demo for nested list

### PR DESCRIPTION
**Pull request by lukasoppermann**
also separate styles classes & js functionality, this makes it more easy to add new items without having to deal with styling issues.

### CODE REVIEW

File: examples/examples.html [hunk 1]
```diff
@@ -53,16 +51,12 @@
 			background: #FEE25F;
 		}
 		#connected {
-			width: 440px;
+			width: 620px;
```
Comment (jacobsvante) [inline examples/examples.html]: Would be nice if you could change this to `max-width` to make stuff responsive.

Comment (lukasoppermann) [inline examples/examples.html]: Done.

Comment (jacobsvante) [inline examples/examples.html]: :+1:

**Comment by lukasoppermann** [comment]
Added another test-case for a disabled list.

**Pull request merged**